packages/node-experimental/src/sdk/trace.ts

```diff
@@ -136,6 +100,22 @@ function getTracer(): Tracer | undefined {
   return client && client.tracer;
 }
 
-function isTransaction(span: Span): span is Transaction {
-  return span instanceof Transaction;
+function _initSpan(span: OtelSpan, spanContext: NodeExperimentalSpanContext): void {
```

Comment:
l: wdyt about calling this something around `applySentryAttributesToSpan` or similar?

Reply:
sounds good to me! 👍